.github/ISSUE_TEMPLATE/---report-a-bug.md

```diff
@@ -0,0 +1,40 @@
+---
+name: "\U0001F41B Report a bug"
+about: Create a report to help us fix bugs
+title: ''
+labels: bug
+assignees: drahnr
+
+---
+
+**Describe the bug**
+
+<!-- A clear and concise description of what the bug is. -->
+
+**To Reproduce**
+
+Steps to reproduce the behaviour:
+
+1. Setting up preconditions with `...`
+2. Run `cargo ...`
+3. Execute `...`
+4. ...
+
+**Expected behavior**
+
+<!-- A clear and concise description of what you expected to happen. -->
+
+**Screenshots**
```

Comment:
I'd encourage against screenshots as it makes it harder to copy parts of it. Maybe a codeblock instead?

Reply:
Sometimes screenshots can capture what codeblocks can't. Perhaps we can ask for both?